src/file.coffee

```diff
@@ -28,7 +28,7 @@ class File
   Section: Construction
   ###
 
-  # Public: Creates a new file.
+  # Public: Configures a new File instance, no files are accessed.
```

Comment:
Actually, upon further inspection, this is not true, as the following is called in the constructor:

```
    if fs.isDirectorySync(filePath)
      error = new Error("#{filePath} is a directory")
      error.code = 'EISDIR'
      throw error
```

Is there a specific reason why the `File` constructor touches the filesystem while the `Directory` constructor does not?